src/client/Microsoft.Identity.Client/Platforms/Features/DefaultOSBrowser/HttpListenerInterceptor.cs

```diff
@@ -89,6 +89,13 @@ public HttpListenerInterceptor(ICoreLogger logger)
                 cancellationToken.ThrowIfCancellationRequested();
 
                 // if cancellation was not requested, propagate original ex
```

Comment:
>// if cancellation was not requested, propagate original ex [](start = 16, length = 59)

i would move this comment to line 99 before the throw.

Reply:
Done.